mpas_analysis/ocean/climatology_map_antarctic_melt.py

```diff
@@ -138,47 +137,128 @@ def __init__(self, config, mpasClimatologyTask, regionMasksTask,
                 f'{observationsDirectory}/Paolo/Paolo_2023_' \
                 f'iceshelf_melt_rates_1992-2017_v1.0_6000x6000km_{res:g}km_' \
                 f'Antarctic_stereo.20240220.nc'
-            refFieldName = 'meltRate'
-            outFileLabel = 'meltPaolo'
-            galleryName = 'Observations: Paolo et al. (2023)'
 
             remapObservationsSubtask = RemapObservedAntarcticMeltClimatology(
                 parentTask=self, seasons=seasons, fileName=obsFileName,
-                outFilePrefix=refFieldName,
+                outFilePrefix='meltRate',
                 comparisonGridNames=comparisonGridNames)
             self.add_subtask(remapObservationsSubtask)
             diffTitleLabel = 'Model - Observations'
 
         else:
             remapObservationsSubtask = None
             controlRunName = controlConfig.get('runs', 'mainRunName')
-            galleryName = None
             refTitleLabel = f'Control: {controlRunName}'
+            diffTitleLabel = 'Main - Control'
+
+        totalFluxVar = 'timeMonthly_avg_landIceFreshwaterFluxTotal'
+        landIceFluxVar = 'timeMonthly_avg_landIceFreshwaterFlux'
+        frazilFluxVar = 'timeMonthly_avg_frazilIceFreshwaterFlux'
 
+        mpasFieldName = totalFluxVar
+
+        if controlConfig is None:
+            refFieldName = 'meltRate'
+        else:
             refFieldName = mpasFieldName
-            outFileLabel = 'melt'
-            diffTitleLabel = 'Main - Control'
 
         for comparisonGridName in comparisonGridNames:
             for season in seasons:
                 # make a new subtask for this season and comparison grid
-                subtask = PlotClimatologyMapSubtask(
+                subtaskName = f'plot_total_melt_{season}_{comparisonGridName}'
+                subtask = PlotAntarcticMeltSubtask(
                     self, season, comparisonGridName, remapClimatologySubtask,
-                    remapObservationsSubtask, controlConfig=controlConfig)
+                    remapObservationsSubtask, controlConfig=controlConfig,
+                    subtaskName=subtaskName)
 
                 subtask.set_plot_info(
-                    outFileLabel=outFileLabel,
-                    fieldNameInTitle='Melt Rate',
+                    outFileLabel='antMeltTotal',
+                    fieldNameInTitle='Total Melt Flux',
                     mpasFieldName=mpasFieldName,
                     refFieldName=refFieldName,
                     refTitleLabel=refTitleLabel,
                     diffTitleLabel=diffTitleLabel,
-                    unitsLabel=r'm a$^{-1}$',
-                    imageCaption='Antarctic Melt Rate',
+                    unitsLabel=r'm a$^{-1}$ freshwater equiv.',
```

Comment:
@xylar Do you mind adding `freshwater equiv.` to the y-axis label of the time series plots for clarity and consistency? I like this addition.

Reply:
Done.